tlz/nofips.go

```diff
@@ -1,4 +1,4 @@
-//go:build !goexperiment.opensslcrypto
+//go:build !goexperiment.opensslcrypto && !requirefips
```

Comment:
both or neither fips-related tags must be present, or the build will fail because tlz.FipsEnable() is not defined